<!DOCTYPE html>
<html lang="en"><head><meta charset="utf-8"><meta content="width=device-width, initiali-scale=1.0, user-scalable=yes" name="viewport"><a href="/rss.xml" rel="alternate" title="David Springate's blog RSS feed" type="application/rss+xml"></a><title>David Springate's blog</title><div class="container-fluid well"><h1>What is this?</h1><p>David Springate's personal blog on programming, data, informatics, biostatistics and evolution</p></div><div class="subnav"><ul class="nav nav-pills"><li><a href="/index.html">Home</a></li><li><a href="/about.html">About</a></li><li><a href="/research.html">Research</a></li><li><a href="/papers.html">Papers</a></li><li><a href="https://github.com/DASpringate">Github</a></li><li><a href="/rss.xml">RSS feed</a></li></ul></div><link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css"><link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css"><link href="/css/smartphone.css" media="only screen and (max-device-width:480px)" rel="stylesheet" type="text/css"></head><body><div class="container-fluid"><div class="row-fluid"><div class="span1"></div><div class="span10"><h1>Research articles</h1><ul><li><p><h5><a href="http://arxiv.org/abs/1212.3730">Function-valued Traits in Evolution</a></h5><em>arXiv - Submitted to JRS interface, awaiting response</em> (2012)</p><p>Pantelis Z. Hadjipantelis, Nick S. Jones, John Moriarty, David A. Springate, Christopher G. Knight</p><p><h6>Abstract: </h6>Many biological characteristics of evolutionary interest are not scalar variables but continuous functions. Given a dataset of function-valued traits generated by evolution, we develop a practical statistical approach to infer ancestral function-valued traits, and estimate the generative evolutionary process. We do this by combining dimension reduction and phylogenetic Gaussian process regression, a non-parametric procedure which explicitly accounts for known phylogenetic relationships. We test the methods' performance on simulated function-valued data generated from a stochastic evolutionary model. The methods are applied assuming that only the phylogeny and the function-valued traits of taxa at its tips are known. Our method is robust and applicable to a wide range of function-valued data, and also offers a phylogenetically aware method for estimating the autocorrelation of function-valued traits.</p><p></li><li><p><h5><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1755-263X.2012.00297.x/abstract">Evaluating the effects of common-pool resource institutions and market forces on species richness and forest cover in Ecuadorian indigenous Kichwa communities</a></h5><em>Conservation Ecology</em> (2012)</p><p>Johan A. Oldekop, Anthony J. Bebbington, Karl Hennermann, Julia McMorrow, David A. Springate, Bolier Torres, Nathan K. Truelove, Niklas Tysklind, Santiago Villamarín, Richard F. Preziosi</p><p><h6>Abstract: </h6>We compare conservation outcomes between a protected area (PA) and four indigenous common-property regimes (CPRs) under differing degrees of market integration in the Ecuadorian Amazon. We first assess how market forces and common-pool resource institutions governing processes of forest conversion affect biodiversity and forest cover, and whether institutions mitigate the effect of market forces. We then analyze how biodiversity and forest cover differ between a PA, and communities with different market access. Finally, we link biodiversity and forest cover changes within communities to differences in land-use practices. While we show similar levels of forest cover and biodiversity between the PA and large CPRs with little access to local markets, institutions appear not to attenuate market effects on conservation outcomes in our case studies. We discuss results within a common-property theory context and highlight the importance of disentangling how market integration, common-pool resource institutions, and resource health interact over time.</p><p></li><li><p><h5><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1420-9101.2011.02360.x/abstract">Correlated response in plasticity to selection for early flowering in Arabidopsis thaliana</a></h5><em>Journal of Evolutionary Biology</em> (2011)</p><p>David Springate, Nora Scarcelli, Jennifer Rowntree, Paula Kover</p><p><h6>Abstract: </h6>Phenotypic plasticity is an important strategy for coping with changing environments. However, environmental change usually results in strong directional selection, and little is known empirically about how this affects plasticity. If genes affecting a trait value also affect its plasticity, selection on the trait should influence plasticity. Synthetic outbred populations of Arabidopsis thaliana were selected for earlier flowering under simulated spring- and winter-annual conditions to investigate the correlated response of flowering time plasticity and its effect on family-by-environment variance (Vg×e) within each selected line. We found that selection affected plasticity in an environmentally dependent manner: under simulated spring-annual conditions, selection increased the magnitude of plastic response but decreased Vg×e; selection under simulated winter-annual conditions reduced the magnitude of plastic response but did not alter Vg×e significantly. As selection may constrain future response to environmental change, the environment for crop breeding and ex situ conservation programmes should be carefully chosen. Models of species persistence under environmental change should also consider the interaction between selection and plasticity.</p><p></li><li><p><h5><a href="http://www.nature.com/nature/journal/v441/n7090/full/nature04566.html">Sympatric speciation in palms on an oceanic island</a></h5><em>Nature</em> (2006)</p><p>Vincent Savolainen, Marie-Charlotte Anstett, Christian Lexer, Ian Hutton, James J. Clarkson, Maria V. Norup, Martyn P. Powell, David A. Springate, Nicolas Salamin, William J. Baker</p><p><h6>Abstract: </h6>The origin of species diversity has challenged biologists for over two centuries. Allopatric speciation, the divergence of species resulting from geographical isolation, is well documented1. However, sympatric speciation, divergence without geographical isolation, is highly controversial2. Claims of sympatric speciation must demonstrate species sympatry, sister relationships, reproductive isolation, and that an earlier allopatric phase is highly unlikely1. Here we provide clear support for sympatric speciation in a case study of two species of palm (Arecaceae) on an oceanic island. A large dated phylogenetic tree shows that the two species of Howea, endemic to the remote Lord Howe Island, are sister taxa and diverged from each other well after the island was formed 6.9 million years ago3. During fieldwork, we found a substantial disjunction in flowering time that is correlated with soil preference. In addition, a genome scan4, 5 indicates that few genetic loci are more divergent between the two species than expected under neutrality, a finding consistent with models of sympatric speciation involving disruptive/divergent selection2. This case study of sympatric speciation in plants provides an opportunity for refining theoretical models on the origin of species, and new impetus for exploring putative plant and animal examples on oceanic islands.</p><p></li><li><p><h5><a href="http://www.sciencedirect.com/science/article/pii/S1055790303003099">Phylogeny and evolution of basils and allies (Ocimeae, Labiatae) based on three plastid DNA regions</a></h5><em>Molecular Phylogenetics and Evolution</em> (2004)</p><p>Alan J Paton,  David Springate, Somran Suddee, Donald Otieno, Renée J Grayer, Madeline M Harley, Fiona Willis, Monique S.J Simmonds, Martyn P Powell, Vincent Savolainen</p><p><h6>Abstract: </h6>A phylogeny of basils and allies (Lamiaceae, tribe Ocimeae) based on sequences of the trnL intron, trnL–trnF intergene spacer and rps 16 intron of the plastid genome is presented. Several methods were used to reconstruct phylogenies and to assess statistical support for clades: maximum parsimony with equally and successively weighted characters, bootstrap resampling, and Bayesian inference. The phylogeny is used to investigate the distribution of morphological, pericarp anatomy, chemical, and pollen characters as well as the geographical distribution of the clades. Tribe Ocimeae is monophyletic and easily diagnosable with morphological synapomorphies. There are monophyletic clades within Ocimeae that broadly correspond to currently recognised subtribes: Lavandulinae, Hyptidinae, Ociminae, and Plectranthinae. Only Lavandulinae has clear non-molecular synapomorphies. Several currently recognised genera are not monophyletic. Floral morphology consistent with sternotribic pollination is most common in Ocimeae, but there are independent departures from this model. Buzz pollination is likely in some species, the only postulated occurrence of this within Lamiaceae. Quinone diterpenoids and flavones in the leaf exudates differ in their distributions across the phylogeny and this could contribute to differences in the recorded medicinal as well as pesticidal uses of the species in the different clades. Mapping geographic distribution on to an ultrametric phylogenetic tree produced using non-parametric rate smoothing supports an Asiatic origin for Ocimeae. There are several secondary occurrences in Asia arising from the African Ociminae and Plectranthinae clades. Colonisation of Madagascar occurred at least five times, and New World colonisation occurred at least three times.</p><p></li></ul></div><div class="span1"></div></div><div class="span12"><div class="span2"></div><div class="span8"><footer class="footer"><p class="right back_to_top"><a href="#">&uArr; Page Top</a></p><p><a href="http://twitter.com/datajujitsu">@datajujitsu</a>&nbsp; 2012</p></footer></div></div><script src="/js/prettify.js" type="text/javascript"></script><script src="/js/lang-clj.js" type="text/javascript"></script><script src="/js/main.js" type="text/javascript"></script></div></body></html>